lib/common.bash

```diff
@@ -167,12 +167,11 @@ clean_env()
 
 clean_env_ctr()
 {
-	check_containers=$(sudo ctr c list -q | wc -l)
-	if ((${check_containers})); then
-		sudo ctr tasks kill $(sudo ctr task list -q)
-		sudo ctr tasks rm -f $(sudo ctr task list -q)
-		sudo ctr c rm $(sudo ctr c list -q)
-	fi
+	local containers=( $(sudo ctr c list -q) )
+	for i in "${containers[@]}"; do
+		sudo ctr task kill -s SIGKILL $(sudo ctr task ls | grep "$i" | awk '{print $1}')
```

Comment:
I wonder if `sudo ctr tasks rm ....` is still needed, why did you remove it?

Reply:
The command` ctr tasks rm` is not required to close and delete containers properly. Additionally that line is causing the non-zero exit.

Reply:
ok, got it, thanks